lwotg/flows.go

```diff
@@ -113,7 +115,13 @@ func (s *Server) stopTraffic() {
 	s.tgMu.Lock()
 	defer s.tgMu.Unlock()
 
+	var wg sync.WaitGroup
 	for _, c := range s.generatorChs {
-		c.Stop <- struct{}{}
+		wg.Add(1)
+		go func(stop chan struct{}) {
+			defer wg.Done()
+			stop <- struct{}{}
+		}(c.Stop)
 	}
+	wg.Wait()
```

Comment:
nit: Could we use a errgroup here instead?

Reply:
Here you mean switching the use of the `chan struct{}` that we use for stop for an `errgroup` that has a `context.Context` which can be cancelled?

I didn't use an `errgroup` here originally because none of these methods return an error currently, they're off trying to generate traffic (or receive it) and then we just want to stop them. The reason that they don't return an error is that there's not really anyone listening (the OTG API is essentially "Start traffic", "Stop traffic" as separate unary RPCs). Definitely open to discussing alternate designs here though.